lib/pages/tasks/bloc/task_bloc.dart

```diff
@@ -18,8 +18,8 @@ class TaskBloc implements BlocBase {
   StreamController<int> _cmdController = StreamController<int>.broadcast();
 
   TaskDB _taskDb;
-  List<Tasks> _tasksList;
-  Filter _lastFilterStatus;
+  late List<Tasks> _tasksList;
+  Filter? _lastFilterStatus;
```

Comment:
We can make Filter non-nullable by giving `filterTodayTasks()` value

Reply:
I did check the code, we already calling filterTodayTasks() in the constructor. Making it late Filter will work.